.github/workflows/go.yml

```diff
@@ -25,9 +25,47 @@ jobs:
       working-directory: ./api-gateway
       run: |
           ls -la
+    
+    - name: Build
+      working-directory: ./audit-service
+      run: |
+          ls -la
+
+
+    - name: Build
+      working-directory: ./blob-service
+      run: |
+          ls -la      
+
+    - name: Build
+      working-directory: ./cd-service
+      run: |
+          ls -la      
+
+    - name: Build
+      working-directory: ./ci-service
+      run: |
+          ls -la
+
+    - name: Build
+      working-directory: ./fabric-service
+      run: |
+          ls -la   
+          
+    - name: Build
+      working-directory: ./monitoring-service
+      run: |
+          ls -la
+
+    - name: Build
+      working-directory: ./notify-service
+      run: |
+          ls -la
+          
```

Comment:
you are just running ls -la over and over again ls list's the content of the directory! and you need to make a seprate yaml file for every service and not make a multi step single file.